data/test_config/stack_config.yml

```diff
@@ -8,3 +8,4 @@ pipeline:
 # https://rasa.com/docs/rasa/policies
 policies:
   - name: MemoizationPolicy
+  - name: RulePolicy
```

Comment:
this was needed to work with the form which I added to the default domain.

Reply:
the presence of the `RulePolicy` in turn led to fallback, which is why I had to add the new story